config-helper.js

```diff
@@ -50,6 +50,33 @@ module.exports = {
 			this.replaceMain({entry: config.entry[opts.chunk || 'main']}, replacement, opts);
 		}
 	},
+	replaceEntry: function (config, replacement, opts = {}) {
+		if (typeof replacement === 'string') {
+			try {
+				replacement = JSON.parse(replacement);
+			} catch (e) {
+				replacement = JSON.parse('{"main": "' + replacement + '"}');
```

Comment:
Should we do this?
I think below will work. If not, let me know.
```suggestion
				newEntry = {main: replacement};
```

Reply:
OK